Packs/MandiantAdvantageThreatIntelligence/Integrations/ThreatIntelligence/ThreatIntelligence.yml

```diff
@@ -0,0 +1,326 @@
+  display: Feed Expiration Policy
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  required: false
+  type: 17
+  additionalInfo: The Feed Expiration Policy
+- name: feedExpirationInterval
+  display: Feed Expiration Interval
+  required: false
+  type: 1
+  additionalInfo: The Feed Expiration Interval
+- defaultvalue: '240'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+  additionalInfo: The number of minutes between Feed Fetches
+- name: feedMinimumConfidence
+  type: 0
+  display: Feed Minimum Confidence Score
+  required: true
+  defaultvalue: 80
+  additionalinfo: 'The minimum MScore value to import as part of the feed'
+- name: feedExcludeOSIntel
+  display: Feed Exclude Open Source Intelligence
+  type: 8
+  required: true
+  defaultvalue: true
+  additionalinfo: 'Whether to exclude Open Source Intelligence as part of the feed'
+- additionalinfo: 'The type of indicators to fetch. Indicator type might include the
+    following: Domains, IPs, Files and URLs.'
+  defaultvalue: Malware,Actors,Indicators
+  display: Mandiant indicator type
+  name: type
+  options:
+  - Malware
+  - Actors
+  - Indicators
+  required: false
+  type: 16
+- additionalinfo: The maximum value allowed is 90 days.
+  defaultvalue: 30 days
+  display: First fetch time
+  name: first_fetch
+  required: false
+  type: 0
+- defaultvalue: '50'
+  display: Maximum number of indicators per fetch
+  name: max_fetch
+  required: false
+  type: 0
+  additionalinfo: Maximum value of 1000.  Any values higher will be capped to 1000
+- additionalinfo: Supports CSV values.
+  display: Tags
+  name: feedTags
+  required: false
+  type: 0
+- additionalinfo: API calls timeout.
+  defaultvalue: '60'
+  display: Timeout
+  name: timeout
+  required: false
+  type: 0
+- additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
+- additionalinfo: Retrieve additional information for each indicator. Note that this
+    requires additional API calls.
+  defaultvalue: 'false'
+  display: Retrieve indicator metadata
+  name: indicatorMetadata
+  required: false
+  type: 8
+- additionalinfo: Note that this requires additional API calls.
+  defaultvalue: 'false'
+  display: Create relationships
+  name: indicatorRelationships
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: 'Integrate with Mandiant Advantage'
+display: Mandiant Advantage Threat Intelligence
+name: Mandiant Advantage Threat Intelligence
+script:
+  commands:
+  - name: threat-intelligence-get-indicators
```

Comment:
Please add the context outputs for the commands as I mentioned above, you can see more documentation about this [here](https://xsoar.pan.dev/docs/integrations/context-and-outputs) and [here](https://xsoar.pan.dev/docs/integrations/context-standards-recommended)